cmd/which-update.rb

```diff
@@ -19,7 +19,7 @@ def which_update_args
       switch "--commit",
              description: "commit the changes using `git`."
       conflicts "--stats", "--commit"
-      max_named 1
+      named_args max: 1
```

Comment:
Can you also change lines 11-15 to be:

```ruby
      description <<~EOS
        Database update for `brew-which-formula`
      EOS
```

And:

```suggestion
      named_args :database, max: 1
```

This will allow the usage string to be automatically generated.